configuration/sitemaps.md

```diff
@@ -471,33 +481,46 @@ mappings=[15="Gone", 19="Chilly", 21="Cozy"]
-By using a Switch or Selection element with a mappings array, you can replace these meaningless values by meaningful descriptions for your inputs in the user interface.
+Imagine your TV is part of your openHAB setup.
+Its power state is represented by a binary Switch Item.
+Its channel number is a discrete number Item that may only be set to one of three states.
+By using a Switch or Selection element with a mappings array, you can replace these meaningless values with user-friendly descriptions for display on the user interface.
+Using this mapping, the power state of the TV, instead of saying "ON" and "OFF" would more accurately display "on" and "standby".
+Similarly, mapping above changes the numbers "1", "2", and "3" to "DasErste", "BBC One", and "Cartoon Network" respectively.
 
-In the third and forth line only a subset of the possible values of items belonging to a heating system is presented to the user.
-This limits the possible input values, which is yet another often occurring use case.
+In the third and forth examples above, only a subset of the possible values of items belonging to a heating system are presented to the user.
+This limits the possible input values, which is yet another often occurring use case for mappings.
 
 ## Dynamic Sitemaps
 
-All Sitemap elements can be configured to be hidden, color highlighted or to have a dynamic icon, depending on certain Item states.
+All Sitemap elements can be configured to be hidden, color highlighted or to have a [dynamic icon]({{base}}/items.md#icons-dynamic), depending on certain Item states.
 A few practical use cases are:
 
-- Show a battery warning if the voltage level of a device is below 30%.
-- Hide further control elements for the TV if it is turned off.
-- Highlight a value with a warning color if it is outside accepted limits.
-- Present a special icon, depending on the state of an item.
+- Show a battery warning if the voltage level of a device is below 30%
+- Hide further control elements for the TV if it is turned off
+- Highlight a value with a warning color if it is outside accepted limits
+- Present a special icon, depending on the state of an item (a [dynamic icon]({{base/items.md#icons-dynamic}}))
 
 ### Visibility
 
-To dynamically show or hide an item, the `visibility` parameter is used.
-By default, an Item is visible if the `visibility` parameter is not provided.
+The `visibility` parameter is used to dynamically show or hide an Item.
+If the parameter is not provided, the default is to display the Item.
+
+Visibility syntax:
 
 ```perl
 visibility=[item_name operator value, item_name operator value, ... ]
 ```
 
-The format of the `visibility` parameter is given above, let's also look at a few examples:
+Valid comparison operators are:
+
+- equal to `==`, unequal to `!=`
+- smaller or equal to `<=`, bigger or equal to`>=`
+- smaller than `<`, bigger than `>`
+
+Expressions are evaluated from left to right.
```

Comment:
Does this one has to be it's own paragraph?

Reply:
Not at all - removed extra line above and below the bullet points.

Reply:
This comment has been addressed.